src/test/regress/Makefile

```diff
@@ -48,6 +50,29 @@ check: check-full
 # check-full triggers all tests that ought to be run routinely
 check-full: check-multi check-multi-mx check-multi-task-tracker-extra check-worker check-follower-cluster check-failure
 
+
+ISOLATION_DEPDIR=.deps/isolation
+ISOLATION_BUILDDIR=build/specs
+
+# this can be used to print a value of variable
+# ex: make print-generated_isolation_files
+print-%  : ; @echo $* = $($*)
+
+.PHONY: create-symbolic-link
+
+create-symbolic-link:
+	mkdir -p $(citus_abs_srcdir)/build
+	ln -fsn $(citus_abs_srcdir)/expected $(citus_abs_srcdir)/build/
+
+$(generated_isolation_files): $(citus_abs_srcdir)/build/specs/%: $(citus_abs_srcdir)/spec/% create-symbolic-link
```

Comment:
the dependency here on `create-symbolic-link` doesn't really make sense. Also, because it's a `.PHONY` target I believe it will always cause a rebuild of the generated_isolation_files.

A better solution would be to do something like this:
```make
isolation_test_files=$(generated_isolation_files) create-symbolic-link
......
# Use this instead of generated_isolation_files
check-isolation: all tempinstall-main $(isolation_test_files)
```